calico_node/startup/startup.go

```diff
@@ -510,10 +535,9 @@ func configureIPPools(client *client.Client) {
 // simplistic check of /proc/sys/net/ipv6 (since platforms that do not have IPv6
 // compiled in will not have this entry).
 func ipv6Supported() bool {
-	// First check the Felix parm.
-	switch strings.ToLower(os.Getenv("FELIX_IPV6SUPPORT")) {
-	case "false", "0", "no", "n", "f":
-		log.Info("IPv6 support disabled through environment")
+	// First check if Felix param is false
+	outgoingNATEnabled := evaluateENVBool("FELIX_IPV6SUPPORT", false)
```

Comment:
This outgoingNATEnabled is not an appropriate name.
I don't think the default was false previously.  If FELIX_IPV6SUPPORT was not set then I think the value would have been the empty string which would not match one of those cases.